pkg/console/metrics/metrics.go

```diff
@@ -30,3 +35,48 @@ func Register() *ConsoleMetrics {
 	})
 	return singleton
 }
+
```

Comment:
This seems a much better abstraction.  All the metrics logic in the metrics package.  I think this may handle all of the cases.

@jhadvig @spadgett for thoughts, anything missing?

Reply:
I could potentially add a `klog.V(4)` message to each.

Reply:
Updated with logs. 2 cases (unitialized, and unhandled) are error cases. Rest, logging if bump the `-v` flag to `4+`.